src/gui/newwizard/pages/accountconfiguredwizardpage.cpp

```diff
@@ -26,28 +26,18 @@ AccountConfiguredWizardPage::AccountConfiguredWizardPage(
     _ui->localDirectoryLineEdit->setText(QDir::toNativeSeparators(userChosenSyncTargetDir));
     _ui->syncEverythingRadioButton->setChecked(true);
 
-    // could also make it invisible, but then the UX is different for different installations
-    // this may be overwritten by a branding option (see below)
-    _ui->useVfsRadioButton->setEnabled(vfsIsAvailable);
+    _ui->useVfsRadioButton->setVisible(vfsIsAvailable);
```

Comment:
There is no apparent reason for this UX change. Please explain. We discussed this in depth when we introduced the new wizard and decided against hiding UX elements randomly on different platforms.

Reply:
Now that we don't have experimental features we basically have https://github.com/owncloud/client/pull/11365/files#diff-d101d06821ce246c78094f95aa54c84df2527a1187475bebf32b7f6cbef29f55L38

Reply:
Yeah, but before, this was only hidden as it was an experimental feature. Now it is just an un_available_ feature. Thus, the button should just be disabled, not hidden.

Reply:
The button was only visible vanilla builds as experimental features where disabled everywhere else.
So we have the exact same behaviour as before.